server/src/main/java/org/elasticsearch/synonyms/SynonymsManagementAPIService.java

```diff
@@ -162,14 +165,7 @@ public void putSynonymsSet(String resourceName, SynonymRule[] synonymsSet, Actio
         // TODO Add synonym rules validation
 
         // Delete synonyms set if it existed previously. Avoid catching an index not found error by ignoring unavailable indices
```

Comment:
Cleanup nit - dup comment to line 219?

Reply:
Thanks, I did not clean up after refactoring 👍